zero-module.yml

```diff
@@ -90,6 +90,12 @@ parameters:
     options:
       - "yes"
       - "no"
+  - field: graphql
+    label: Enable graphql and apollo server?
```

Comment:
How about changing this to `apiType`, the label to "What type of API do you want to expose?" and have the options be "graphql" and "rest"?

Reply:
sounds better. I am changing it.

Reply:
Solved